assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
+function search_card() {
+  let input = document.getElementById('searchbar').value
+  input=input.toLowerCase();
+  let x = document.getElementsByClassName('card');
+    
+  for (i = 0; i < x.length; i++) { 
+      if (!x[i].innerHTML.toLowerCase().includes(input)) {
```

Comment:
We should change `innerHTML` to `textContent`. The first returns all the tags and their attributes so we would be searching for matches that the user doesn't need and it confuses the result. The second only returns the text without the html so the search will be more accurate. (Don't forget to change variable names)

` if (!cards[i].textContent.toLowerCase().includes(value)) {`